include/sys/bqueue.h

```diff
@@ -27,10 +27,14 @@ extern "C" {
 
 typedef struct bqueue {
 	list_t bq_list;
+	size_t bq_size;
+	list_t bq_deqing_list;
+	size_t bq_deqing_size;
+	list_t bq_enqing_list;
+	size_t bq_enqing_size;
```

Comment:
Could we change enqing and deqing to enqueuing and dequeuing respecctively?

deqing and enqing are not English words and seeing them causes my brain switch to thinking that I am reading Chinese Pingyin. That makes these variable names harder to read for me (and possibly others with basic knowledge of pingyin). I probably can adapt, but I would prefer to read something that is more mnemonic be default.

Reply:
sure thing.